endpoints/api/billing.py

```diff
@@ -1040,7 +1079,7 @@ def get(self):
         user = get_authenticated_user()
         account_number = marketplace_users.get_account_number(user)
         if not account_number:
-            raise NotFound()
+            return []
```

Comment:
Likewise, why go from 404 to 200? The client should be able to handle this.
An empty list should be returned in the case where the account number exists, but no subscriptions is attached to that user

Reply:
Same reason as above, the billing ui would look like it was indefinitely loading when the api returned the 404. I found a way to properly handle the frontend though and updated the response codes accordingly